packages/arb-rpc-node/batcher/sequencerBatcher.go

```diff
@@ -395,6 +408,58 @@ func (b *SequencerBatcher) createBatch(ctx context.Context, newMsgCount *big.Int
 		lastAcc = item.Accumulator
 	}
 
+	delayBlocks := new(big.Int).Sub(b.latestChainTime.BlockNum.AsInt(), l1BlockNumber)
+	delaySeconds := new(big.Int).Sub(b.latestChainTime.Timestamp, l1Timestamp)
+	if delayBlocks.Cmp(b.maxDelayBlocks) > 0 || delaySeconds.Cmp(b.maxDelaySeconds) > 0 {
+		logger.Error().Str("delayBlocks", delayBlocks.String()).Str("delaySeconds", delaySeconds.String()).Msg("Exceeded max sequencer delay! Reorganizing to compensate...")
+		b.inboxReader.MessageDeliveryMutex.Lock()
+		defer b.inboxReader.MessageDeliveryMutex.Unlock()
+
+		newestMsgCount, err := b.db.GetMessageCount()
+		if err != nil {
+			return false, err
+		}
+		if newestMsgCount.Cmp(newMsgCount) > 0 {
+			newerBatchItems, err := b.db.GetSequencerBatchItems(newMsgCount, newestMsgCount)
+			if err != nil {
+				return false, err
+			}
+			batchItems = append(batchItems, newerBatchItems...)
+		}
+
+		var previousSeqBatchAcc common.Hash
+		if prevMsgCount.Cmp(big.NewInt(1)) > 0 {
```

Comment:
Why is this greater than 1 instead of greater than 0?

Reply:
Interesting, I copied this from deliverDelayedMessages, which was also apparently broken haha. I'll fix them both.